stemcell_builder/stages/base_ssh/apply.sh

```
@@ -24,6 +24,9 @@ echo 'X11Forwarding no' >> $chroot/etc/ssh/sshd_config
 sed "/^ *MaxAuthTries/d" -i $chroot/etc/ssh/sshd_config
 echo 'MaxAuthTries 3' >> $chroot/etc/ssh/sshd_config
 
+sed "/^ *MaxStartups/d" -i $chroot/etc/ssh/sshd_config
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)